elasticsearch/templates/statefulset.yaml

```diff
@@ -111,6 +111,14 @@ spec:
           configMap:
             name: {{ template "uname" . }}-config
         {{- end }}
+{{ if .Values.keystore }}
+        - name: keystore
+          emptyDir: {}
+{{ end }}
+        {{- range .Values.keystore }}
```

Comment:
curious, why not have this `{{- range ...}}` inside the `{{ if ... ` block above? I I don't think there'd be any behavior changes, just curious about the style choice.

Reply:
This is just a leftover from the original refactoring. But you are right that it doesn't change anything. I fixed it up to make it as explicit as possible.